anomalib/data/mvtec.py

```diff
@@ -259,6 +260,7 @@ def __getitem__(self, index: int) -> Dict[str, Union[str, Tensor]]:
             label_index = self.samples.label_index[index]
 
             item["image_path"] = image_path
+            item["class"] = label
```

Comment:
Maybe instead of "class" we could call this "subclass", "anomalous_subclass", "defect_type" or something similar. This would avoid confusion.

Reply:
maybe `anomaly_type`? I also like `defect_type`, but the content of the task may not always be defect detection.

Reply:
or in simple terms, just `sub_class`